projects/cds/src/merchandising/facade/cds-merchandising-product.service.spec.ts

```diff
@@ -10,6 +10,7 @@ import { CdsMerchandisingSiteContextService } from './cds-merchandising-site-con
 import { CdsMerchandisingUserContextService } from './cds-merchandising-user-context.service';
 import createSpy = jasmine.createSpy;
 
+const uuidv4 = require('uuid/v4');
```

Comment:
Can we just mock the UUID, or is there a need to generate a proper one?

Reply:
I've changed it to a string literal